regress/unittests/utf8/tests.c

```diff
@@ -9,7 +9,7 @@
 #include <locale.h>
 #include <string.h>
 
-#include "test_helper.h"
+#include "../test_helper/test_helper.h"
```

Comment:
This looks like Windows specific change. The intent is to try to keep the source same wherever possible or add WINDOWS #def around. In this case, looks like adding test_helper directory to INCLUDE in project file should do the trick?